### ISSUE TITLE
Wanted: reproducer for error "The type is non-fibrant or its sort depends on an interval variable"

### ISSUE THREAD

**Issue by andreasabel** [agda/agda#7413]
https://github.com/agda/agda/blob/6d6d8470de542a91f52a4897c4934de547a363c1/src/full/Agda/TypeChecking/Telescope/Path.hs#L50-L52
@plt-amy: Do you think you could cook up a test case that triggers this error?
It has to be something with `with` because the function `telePiPath` is called from `withFunctionType` only:
https://github.com/agda/agda/blob/6d6d8470de542a91f52a4897c4934de547a363c1/src/full/Agda/TypeChecking/With.hs#L137-L145

**Comment by plt-amy** [comment]
Sure, I'll see if I can come up with something.

**Comment by andreasabel** [comment]
@plt-amy: Could you try, please?

(Or we make this `__IMPOSSIBLE__` and see if someone reports a bug.)

**Comment by plt-amy** [comment]
@andreasabel Not until the hackathon, probably .

**Comment by Saizan** [comment]
I believe this is actually impossible at the moment unless generalized Path types were implemented while I wasn't looking:
`telePathPi` only does this check if there's a boundary, which should only be introduced by a PathP copattern, which then should ensure the result type is in `Type lvl` for some `lvl` that does not depend on on the interval variable of the path.

**Comment by andreasabel** [comment]
Thanks @Saizan for the analysis!

**Comment by andreasabel** [comment]
Looks like this was a duplicate of
- #7226
